FirelandsProject/cataclysm
# Fix (DB/Creature Template) Add flag trainer

### Changes proposed

- The vendor flag is added to the npc, because inside DBErrors, it was mentioned that it was being ignored. When checking and verifying that they were sellers, the seller flag is added to the flag they have, thus eliminating the error.

### Issues addressed


- Closes

### SOURCE:


- Table `npc_trainer` contains an entry for a creature template (Entry: 3703) without trainer flag, ignoring
- Table `npc_trainer` contains an entry for a creature template (Entry: 6387) without trainer flag, ignoring
- Table `npc_trainer` contains an entry for a creature template (Entry: 7232) without trainer flag, ignoring

### Tests performed


- Windows 10
- In-game

### How to Test the Changes:


1. When you turn on the server, you will see a series of errors, which will be running in several pull request, but there is one, which mentions the flag of the flag in these npc.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
